js/index.js

```diff
@@ -1,144 +1,252 @@
+      "Conducted A/B experiments to test our assumptions on user behavior and make more informed product decisions",
+    ],
+  },
 
-function initSkills(container){
-	for( var i in skillSets ){
-		var skillSet = skillSets[i];
+  {
+    title: "Wakebird",
+    link: "https://wakebird.com",
+    metadata: ["Co-Founder", "January 2019 - Present"],
+    tasks: [
+      "Led a two-person team to design, build, and sell a web application for organizing dragon boat racing teams",
+      "Mentored a junior engineer on code review, sprint planning, node.js, and React",
+      "Synthesized interviews with athletes and coaches into meaningful feature design specs",
+      "Taught athletes and coaches how to use Wakebird, and solicited feedback to get to the bottom of customer needs",
+      "Reviewed bookkeping ledger and tax remittance on a quarterly basis",
+    ],
+  },
 
-		var columnDiv = $("<div/>",{
-			"class":"column"
-		}).appendTo(container);
+  {
+    title: "Flatiron Health",
+    link: "https://flatiron.com",
+    metadata: ["Senior Software Engineer", "June 2019 - August 2021"],
+    tasks: [
+      "Drafted and built the infrastructure for a machine-learning-based product that predicts patient ER visits for oncology nurses",
+      "Refactored and maintained complex Extract, Transform & Load (ETL) pipelines with Apache Airflow for quarterly delivery of abstracted cancer data",
+      "Designed UI and wrote the code for a research project tracking dashboard that served as a communication platform between academic researchers and Flatiron clinical data analysts",
+    ],
+  },
 
-		$("<h3/>",{
-			text: skillSet.title
-		}).appendTo(columnDiv);
+  {
+    title: "Knewton",
+    link: null,
+    metadata: ["Software Engineer II", "November 2018 - March 2019"],
+    tasks: [
+      "Investigated slow queries to solve slowdowns on a Django-based Content Management System (CMS).",
+      "Quickly picked up Python, Typescript, and Phabricator concepts",
+      "Expanded test coverage across the CMS, learning more specific technical details about pytest",
+    ],
+  },
 
-		for( var j in skillSet.entries ){
-			var skill = skillSet.entries[j];
+  {
+    title: "Kinvite",
+    link: null,
+    metadata: ["Co-Founder and CTO", "September 2016 - September 2018"],
+    tasks: [
+      "Built the entire stack of Web, Mobile, Backend, and Infrastructure from the ground-up using React, React-Native, Scala/Play, Postgresql, and Google Cloud.",
+      "Hired and led Design, QA, and software freelancers to help contribute to the product",
+      "Built a three-party payment processing system for tickets with Stripe Connect",
+    ],
+  },
 
-			var entryDiv = $("<div/>",{
-				"class":"entry"
-			}).appendTo(columnDiv);
+  {
+    title: "EDDI: A Desalinator for Irrigation",
+    link: "https://www.youtube.com/watch?v=A_Z0X6SA5Hk",
+    metadata: ["General Lead", "May 2015 - September 2017"],
```

Comment:
What's a General Lead?
Team Lead? Technical Lead?

Reply:
Yo nobody gave me any title. We didn't have titles we were just 6 kids loll

Reply:
f it. Team lead.